cimetrics/plot.py

```diff
@@ -100,6 +100,10 @@ def less_than_max_build_id(record):
             .groupby("build_id")
             .mean()
         )
+        # Drop incomplete rows
+        if "__complete" in df.columns:
```

Comment:
Only applying this logic if we have at least one "new-style" build in history is perhaps questionable. It seems better than killing off history in one go?

I think this ought to be removed eventually. In general, if there are no complete builds on a branch, there is effectively nothing!